### DISCUSSION TITLE
Is there a way to not put netlify-identity-widget.js somewhere other than the index?

### DISCUSSION THREAD

**Discussion by saturnonearth** [decaporg/decap-cms#6178] [Q&A]
I don't want to bloat up my index page when the CMS is only used by a handful of people. Is there a way to change the page it's on, or not use it at all?

**Comment by talves** [comment]
You should be able to put it in your `<cms-root>/index.html` file right? I can't remember if it is required by Netlify to be in the root of the site or not.

If it is required to be in the root of the site, you do have the option to pull the cms out to it's own app and make it the root of it's own site. Maybe do some testing for the root of the cms app first and let us know how that goes.

**Reply by saturnonearth** [reply]
In the docs it says it needs to be a the site index, as well as the CMS index, sadly

**Reply by talves** [reply]
I thought I read somewhere it does work. Maybe this will work as @erezrokah explained in this answer. https://answers.netlify.com/t/11ty-how-to-use-netlify-identity-widget-without-lost-of-performance/33175/3?u=talves

Nice explanation of why it uses the root path as a default.

**Reply by talves** [reply]
Sorry for all the link digging to the setup ahead of time 😁

**Reply by talves** [reply]
Also be aware of the links for users already logged into the site if you make this change. That might give them links back to the root which doesn't exist any longer with this change.

For those that have a lot of users and make this change, you might be able to proxy the path to the new path.

**Reply by saturnonearth** [reply]
Thanks for the help!! I was able to get it working by simply removing from index. Not sure if it still works because my users use Google Oauth to log in?

**Reply by tomrutgers** [reply]
The identity emails point to your root by default, so adding it to your index is the most plug & play experience. You can change the identity mails though: https://answers.netlify.com/t/common-issue-netlify-cms-git-gateway-email-not-confirmed/10690

**Reply by saturnonearth** [reply]
Ah I understand. From the documentation it sounded like it was "required", but since I am the only one who is going to use the CMS, it's not a big deal, and didn't need it.
